src/main/kotlin/gg/essential/elementa/components/ScrollComponent.kt

```diff
@@ -800,6 +806,30 @@ class ScrollComponent constructor(
 
     }
 
+    /**
+     * Constraints the scrollbar grip's height to be a certain minimum height, or the [desiredHeight].
+     * This is the default constraint for vertical scrollbar grips if [limitScrollBarGripHeight] is enabled.
+     *
+     * @param desiredHeight The intended height for the scrollbar grip.
+     */
+    inner class ScrollBarGripMinHeightConstraint(
+        private val desiredHeight: HeightConstraint
```

Comment:
This constraint is never updated (`animationFrame`), so it'll only be evaluated once and then won't ever change after that, even when it should. See one of the existing composed constraints (e.g. `CoerceAtLeastConstraint`) for how to do this properly.